### DISCUSSION TITLE
CatBoost Target Statistics

### DISCUSSION THREAD

**Discussion by ReinvanLennep** [pycaret/pycaret#2190] [Q&A]
Hi, 

I am investigating my results from my Catboost model (using Pycaret), however I was wondering where I can find the Target Statistic values (TS) of my categorical features? Are they stored somewhere?

Because it seems like my categorical features are transformed by using one-hot encoding. However, this is not how I expect Catboost to be functioning. 

Thx in advance!

**Comment by KitTarzan** [comment]
Pycaret does one-hot encoding on categorical features automatically, unless they are high cardinality or you override it manually. In those cases you can have for ordinal, frequency, or clustering based encoding.  But no matter what it will do some sort of categorical encoding. It does not interact with the categorical features capabilities of algorithms like LightGBM, XGBoost, or Catboost the way you may expect. Catboost is not going to get a list past to its cat_features parameter

**Comment by m6129** [comment]
@KitTarzan, @ReinvanLennep  
Excuse Me, how do you use Catboost in Pycaret?
"pip install pycaret[full]" does not give the desired result

**Reply by ngupta23** [reply]
Is something wrong with how it is getting installed in your machine? Catboost is included in the optional install as can be seen here:

https://github.com/pycaret/pycaret/blob/4bf7452c3c4129c028728ac42d43d504a5e7cf91/requirements-optional.txt#L13

**Reply by m6129** [reply]
Thanks. Google Colab notebook have this problem. 
There are no problems with this in Kaggle notebook.
